lib/const.rb

```diff
@@ -1,16 +1,19 @@
 # Defines common constants used in different parts of crew
 
-CREW_VERSION = '1.8.9'
+CREW_VERSION = '1.8.10'
 
-ARCH_ACTUAL = `uname -m`.strip
+ARCH_ACTUAL = `uname -m`.chomp
 # This helps with virtualized builds on aarch64 machines
 # which report armv8l when linux32 is run.
-ARCH = if ARCH_ACTUAL == 'armv8l' then 'armv7l' else ARCH_ACTUAL end
+ARCH = ENV['CREW_ARCH'] || if ARCH_ACTUAL == 'armv8l' then 'armv7l' else ARCH_ACTUAL end
```

Comment:
Why would we ever want the architecture in an environment variable?  Shouldn't that go with the territory?

Reply:
To test out multiarching stuff, or to run 32 bit code on a 64 bit machine

Reply:
CREW_ARCH looks useful. I'm literally modifying ARCH in install.sh with sed for all of my installs in containers because I'm running on a RPI4B with 8gb of ram on a 64 bit kernel...

Reply:
If you need to change the architecture then you are probably not operating in the correct environment.